packages/io/dxf-deserializer/README.md

```diff
@@ -1,36 +1,37 @@
 ## @jscad/dxf-deserializer
 
-> DXF deserializer for the JSCAD project
+> Deserializer of DXF data to CSG geometries
 
 [![GitHub version](https://badge.fury.io/gh/jscad%40jscad%2Fdxf-deserializer.svg)](https://badge.fury.io/gh/jscad%40jscad%2Fdxf-deserializer)
 [![Build Status](https://travis-ci.org/jscad/io.svg)](https://travis-ci.org/jscad/dxf-deserializer)
 
 ## Overview
 
-This deserializer converts raw DXF data to JSCAD code fragments, or converts raw DXF data directy to CSG library objects. When converting unknown DXF data, converting to JSCAD code fragments will provide more information.
+This deserializer converts raw DXF formatted data (files) to JSCAD scripts or CSG geometries.
+When converting unknown DXF data, converting to JSCAD scripts will provide more information on the converted DXF entities.
 
 ### DXF Implementation Notes
 
 The [DXF(tm) file structure](http://docs.autodesk.com/ACD/2014/ENU/files/GUID-73E9E797-3BAA-4795-BBD8-4CE7A03E93CF.htm) changes continously as AutoDesk releases new features and fixes. So, you can imagine that after 20 years, the contents of DXF files are pretty messed up. Expect the worst.
 
 **NOTE: At this time, only ASCII DXF files are supported. BINARY DXF files are not supported.**
 
-This deserializer converts only what is required by JSCAD libraries and applications. Full document conversion is NOT supported, so don't even try. However, conversion of the following DXF entities are possible:
+This deserializer converts only what is supported by JSCAD libraries. Full document conversion is NOT supported, so don't even try. However, conversion of the following DXF entities are possible:
```

Comment:
the 'so don't even try part' is kind off odd, tone wise ?

Reply:
Done